README.md

```diff
@@ -1 +1,35 @@
-# shift-left-quality-sample
\ No newline at end of file
+# Welcome to Shift-left-quality discussion
+
```

Comment:
Both headers in same length, I assume second one must be secoundry header, it might need to be changed with H2 level .